sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/CallAutomationClientBuilder.java

```diff
@@ -81,6 +82,7 @@ public final class CallAutomationClientBuilder implements
     private ClientOptions clientOptions;
     private RetryPolicy retryPolicy;
     private RetryOptions retryOptions;
+    private CommunicationUserIdentifier sourceIdentity;
```

Comment:
Seems like this variable is optional. What would happen when the user is creating a call / answering a call / adding a participant if this sourceIdentity is never provided by the user?